client/src/components/App.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import ProductDetailPage from './ProductDetailPage.jsx';
+
+class App extends React.Component {
+  constructor (props) {
+    super (props);
+  }
+
+  render () {
+    return (
+      <Router>
+        <h1>PROJECT ATLIER</h1>
+        <Switch>
+        <Route path="/product/:id" component={ProductDetailPage}/>
```

Comment:
This main App component is mostly used for routing. The original function of getting product info and such has been moved out to facilitate dynamic product loading via react routing.

Reply:
React router is new to me, just read up on it and this looks great. looks like it'll be a lot better than the initial architecture. Excited to use it!

Reply:
This looks great to me. I only got to do React router in the advanced content for Recast.ly Redux, so I am a little fuzzy, but this code is starting to refresh my memory. Thanks a ton for putting this together so quickly!